templates/messaging.patch

```diff
@@ -0,0 +1,174 @@
+diff --git templates/create-daml-app/daml/User.daml templates/create-daml-app/daml/User.daml
```

Comment:
Can we just kill the existing copy now and use this in the docs as well? Otherwise we are now up to having two copies of the new code which doesn’t seem great.

Reply:
Yes, that's the plan :)

Reply:
Done in this PR now